CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - build(ios): Bump sentry-cocoa to 7.0.0 and remove setLogLevel #1459
 - feat: Close method to fully disable the SDK on all layers #1457
+- build(android): Bump Android SDK to 5.0.0-beta.1 #1476
```

Comment:
One question though: Are there any breaking changes worth mentioning in the Changelog?

Reply:
When I did a run through the changelog, I don't think there were any that would apply to RN. Can you please confirm @marandaneto?

Reply:
We agreed nothing that concerns RN so we're good to go here

Reply:
5.0.0-beta.2 will require changes though